omod/src/main/webapp/admin/concepts/conceptProposalForm.jsp

```diff
@@ -134,8 +134,8 @@
 									class='searchHit'>
 									${status.index + 1})
 									<c:choose >
-										<c:when test="${listItem.synonym != ''}">
-											<span class='mainHit'>${listItem.synonym}</span>
+										<c:when test="${listItem.description != ''}">
```

Comment:
From here, https://github.com/openmrs/openmrs-core/blob/1.12.x/web/src/main/java/org/openmrs/web/dwr/ConceptListItem.java#L46-L53 the synonym was deprecated and hence removed. Therefore, just remove the choose and simply use listItem.name which is already in the otherwise section.

Reply:
Thank you. I was viewing the wrong ConceptListItem.java so didn't find this deprecated statement.

Reply:
No. You did not look at the wrong guy. You just looked after it was deprecated and eventually removed. 😊

Reply:
Ah, on an older version, that makes sense.